cpp/CPP14.g4

```diff
@@ -32,7 +32,7 @@ translationunit
 
 
 primaryexpression
-   : literal
+   : literal (literal)*
```

Comment:
Please remove redundant parentheses and use `+` operator:

```suggestion
   : literal+
```

Reply:
Thank you KvanTTT

Reply:
Problems with CPP14.g4 #1577  duplicate issues.

Reply:
What do you mean? I guess `literal (literal)*` and `literal+` are equivalent forms.